akka-remote/src/main/scala/akka/remote/RemoteWatcher.scala

```diff
@@ -32,7 +32,7 @@ private[akka] object RemoteWatcher {
   final case class UnwatchRemote(watchee: InternalActorRef, watcher: InternalActorRef)
 
   @SerialVersionUID(1L) case object Heartbeat extends HeartbeatMessage
-  @SerialVersionUID(1L) final case class HeartbeatRsp(addressUid: Int) extends HeartbeatMessage
+  @SerialVersionUID(1L) final case class HeartbeatRsp(addressUid: Long) extends HeartbeatMessage
```

Comment:
where is serialization of this? I expected that to change also. Or are we using Java serialization?